record-trace/engine/src/commandline.rs

```diff
@@ -116,10 +133,14 @@ impl RecordArgs {
         if !args.on_cpu && !args.off_cpu &&
             !args.soft_page_faults && !args.hard_page_faults &&
             args.script.is_none() {
+            warn!("No events or scripts selected");
             eprintln!("No events or scripts selected. Exiting.");
             process::exit(1);
         }
 
+        debug!("Arguments parsed successfully: format={}, on_cpu={}, off_cpu={}, soft_page_faults={}, hard_page_faults={}, live={}", 
```

Comment:
Missing target_pids and script.  All of the command line arguments should be written out at info level, each on a separate call to log.